README.md

```diff
@@ -1,6 +1,20 @@
 Absolute
-=======
+=============
+
+### A web-based order platform like siren-order for Starbucks.
```

Comment:
Yeah, I respect your opinion. My point was about look-and-feel, not about content.
I did find some examples:
  - swift: https://github.com/apple/swift
  - linux: https://github.com/torvalds/linux
  - flutter: https://github.com/flutter/flutter
  - react: https://github.com/facebook/react

They have each their own style, but I didn't see a case that uses a |bold| style description below the project title. I thought that the consecutive |bold| style might be bad look-and-feel to present our project. But it's a personal taste. So, we can vote it. (for our members)

I agree with your opinion. We still need a short to present our project.
Also, if we have it, we can add it to the topic section (top of this page)